modules/terraform/plan.go

```diff
@@ -7,23 +7,55 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// InitAndPlan runs terraform init and plan with the given options and return stdout/stderr from the apply command.
-func InitAndPlan(t *testing.T, options *Options) int {
-	exitCode, err := InitAndPlanE(t, options)
+// InitAndPlan runs terraform init and plan with the given options and returns stdout/stderr from the plan command.
+func InitAndPlan(t *testing.T, options *Options) string {
+	out, err := InitAndPlanE(t, options)
+	require.NoError(t, err)
+	return out
+}
+
+// InitAndPlanE runs terraform init and plan with the given options and returns stdout/stderr from the plan command.
+func InitAndPlanE(t *testing.T, options *Options) (string, error) {
+	if _, err := InitE(t, options); err != nil {
+		return "", err
+	}
+
+	if _, err := GetE(t, options); err != nil {
+		return "", err
+	}
+
+	return PlanE(t, options)
+}
+
+// Plan runs terraform plan with the given options and returns stdout/stderr.
+func Plan(t *testing.T, options *Options) string {
+	out, err := PlanE(t, options)
+	require.NoError(t, err)
+	return out
+}
+
+// PlanE runs terraform plan with the given options and returns stdout/stderr.
+func PlanE(t *testing.T, options *Options) (string, error) {
+	return RunTerraformCommandE(t, options, FormatArgs(options, "plan", "-input=false", "-lock=false")...)
+}
+
+// InitAndPlanWithExitCode runs terraform init and plan with the given options and returns exitcode for the plan command.
```

Comment:
```suggestion
// InitAndPlanWithExitCode runs terraform init and plan with the given options and returns exitcode for the plan command.
// This will fail the test if there is an error.
```